starknet_py/net/schemas/rpc.py

```diff
@@ -155,6 +169,7 @@ class TypesOfTransactionsSchema(OneOfSchema):
         "DEPLOY": DeployTransactionSchema,
         # FIXME add proper handling/serialization
         "DEPLOY_ACCOUNT": None,
```

Comment:
Do we have an issue for this fixme? If not, let's create one.

Reply:
The issue is to support v0.2.1 😅